Marlin/Configuration.h

```diff
@@ -81,6 +81,20 @@
 // example_configurations/SCARA directory.
 //
 
+//===========================================================================
+//======================== Piston Extruder Printer ==========================
+//===========================================================================
+// For a Piston Extruder printer replace configuration.h with the file in the
+// example_configurations/MMM directory.
+//
+
+//#define PISTON_EXTRUDER //used for the Piston Extruder dryer fan
+//#define E_ADJUST_MANUALLY //variable resistor has one end to ground, other to +5 volts. & wiper to E_ADJUST_PIN
```

Comment:
The naming of this option is perhaps over-generalized. Is this usable without the piston extruder feature?